{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2013-0484",
      "ASSIGNER": "cve@mitre.org"
    },
    "affects": {
      "vendor": {
        "vendor_data": [
          {
            "vendor_name": "ibm",
            "product": {
              "product_data": [
                {
                  "product_name": "cognos_tm1",
                  "version": {
                    "version_data": [
                      {
                        "version_value": "10.1.0"
                      },
                      {
                        "version_value": "10.1.0.1"
                      },
                      {
                        "version_value": "10.1.1"
                      }
                    ]
                  }
                }
              ]
            }
          }
        ]
      }
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "NVD-CWE-noinfo"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "http://www-01.ibm.com/support/docview.wss?uid=swg21637655"
        },
        {
          "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/81612"
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "The server process in IBM Cognos TM1 10.1.x before 10.1.1 FP1 allows remote attackers to cause a denial of service (daemon crash) via an undocumented API call that triggers the transmission of unexpected data."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "operator": "OR",
        "cpe": [
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:ibm:cognos_tm1:10.1.0",
            "cpe23Uri": "cpe:2.3:a:ibm:cognos_tm1:10.1.0:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:ibm:cognos_tm1:10.1.0.1",
            "cpe23Uri": "cpe:2.3:a:ibm:cognos_tm1:10.1.0.1:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:ibm:cognos_tm1:10.1.1",
            "cpe23Uri": "cpe:2.3:a:ibm:cognos_tm1:10.1.1:*:*:*:*:*:*:*"
          }
        ]
      }
    ]
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "version": "2.0",
        "vectorString": "(AV:N/AC:M/Au:N/C:N/I:N/A:P)",
        "accessVector": "NETWORK",
        "accessComplexity": "MEDIUM",
        "authentication": "NONE",
        "confidentialityImpact": "NONE",
        "integrityImpact": "NONE",
        "availabilityImpact": "PARTIAL",
        "baseScore": 4.3
      },
      "severity": "MEDIUM",
      "exploitabilityScore": 8.6,
      "impactScore": 2.9,
      "obtainAllPrivilege": false,
      "obtainUserPrivilege": false,
      "obtainOtherPrivilege": false,
      "userInteractionRequired": false
    }
  },
  "publishedDate": "2013-06-19T13:39Z",
  "lastModifiedDate": "2017-08-29T01:33Z"
}